app/components/Layout.css.ts

```diff
@@ -70,7 +70,7 @@ export const dot = style({
 			borderRadius: "100%",
 			display: "inline-block",
 			height: "0.85rem",
-			margin: "0 1.5rem 1rem",
+			margin: "0 1rem 1rem",
```

Comment:
[Praise] _Yes_ the inconsistency of vertical margin/padding was bugging me...

Reply:
It was more to make the 1-line nav concise, it looked weird all stretched out.